data/serializer/src/main/java/tech/pegasys/teku/api/response/v1/node/GetPeerScoresResponse.java

```diff
@@ -0,0 +1,15 @@
+package tech.pegasys.teku.api.response.v1.node;
```

Comment:
I'd move this to: `tech.pegasys.teku.api.response.v1.teku`

Reply:
Done. Thank you!